include/arch/x86/rte_atomic.h

```diff
@@ -216,7 +216,7 @@ static inline int rte_atomic32_dec_and_test(rte_atomic32_t *v)
 }
 #endif
 
-#ifdef RTE_ARCH_I686
+#if defined(__i686__)
 #include "arch/x86/rte_atomic_32.h"
 #else
```

Comment:
Better to add a check for `__x86_64__`.

Reply:
#if defined(__x86_64__) || defined(__i686__)
#include <arch/x86/rte_pause.h>	#include <arch/x86/rte_pause.h>
#include <arch/x86/rte_atomic.h>	#include <arch/x86/rte_atomic.h>

Is it for better reading? we have checked that only ` __x86_64__` and `__i686__` could include `arch/x86/rte_atomic.h`

Reply:
oh I got it, maybe other files will include `<arch/x86/rte_pause.h>`. I will update it.